test/tools/yulInterpreter/Interpreter.cpp

```diff
@@ -122,7 +138,11 @@ void Interpreter::operator()(Block const& _block)
 			m_functions[funDef.name] = &funDef;
 			m_scopes.back().insert(funDef.name);
 		}
-	ASTWalker::operator()(_block);
+
+	for (auto const& statement: _block.statements)
+		if (m_state.loopState != LoopState::Default)
```

Comment:
Shouldn't it be `==` Default?

Reply:
ah, yes. I didn't finish thinking here. sorry. it was meant to have a `break;` underneath.